test/resource/changes/cascade_destroy_test.exs

```diff
@@ -144,7 +144,10 @@ defmodule Ash.Test.Resource.Change.CascadeDestroy do
     1..Enum.random(3..5)
     |> Enum.map(fn _ -> Post.create!(%{author_id: author.id}) end)
 
-    Ash.bulk_destroy!([author], :no_notification_destroy, %{})
+    Ash.bulk_destroy([author], :no_notification_destroy, %{},
+      return_errors?: true,
+      strategy: [:stream, :atomic, :atomic_batches]
```

Comment:
Needed to add `:stream` because it now calls the before_batch hook and is therefore not atomic anymore

Reply:
Yeah, I forgot to change this back after adding in the old behaviour as default again.